guide/C/ch_currency.xml

```diff
@@ -34,793 +35,1132 @@
+
+        <table>
+          <title>Selling a currency with a Split Transaction Scheme</title>
+
+          <tgroup cols="3">
+	    <colspec colwidth='3*' align='left'/>
+	    <colspec colwidth='1*' align='center'/>
+	    <colspec colwidth='1*' align='center'/>
+	    <thead>
+              <row><entry>Account</entry><entry>Deposit</entry><entry>Withdrawal</entry></row>
+	    </thead>
+            <tbody>
+              <row>
+                <entry>Assets:Investments:Currency:Bank</entry> <entry>Sold Amount - Exchange Fee</entry> <entry></entry> </row>
+              <row> <entry>Expenses:Investments:Currency:Currency Bank:XXX</entry> <entry>Exchange Fee</entry> <entry></entry> </row>
+	      <row> <entry>Assets:Investments:Currency:XXX</entry> <entry></entry> <entry>Sold Amount</entry> </row>
+	      <row> <entry>Income:Investments:Currency Bank:Capital Gains:XXX</entry> <entry>[LOSS]</entry> <entry>PROFIT</entry> </row>
+              <row> <entry>Assets:Investments:Currency:XXX</entry> <entry>PROFIT (with To Amount = 0)</entry> <entry>[LOSS (with To Amount = 0) ]</entry> </row>
+            </tbody>
+          </tgroup>
+        </table>
+
+      </sect3>
+    </sect2>
 
+    <sect2 id="currency_reconcile1">
+      <title>Reconciling Statements in a Foreign Currency</title>
+
+      <para>Reconciling foreign statement are done in the same manner as when
+      you reconcile your local bank statement. If you have created a Chart of
+      Accounts structure which allows you to have the same currency per account
+      as your statement, it is actually exactly the same as
+      reconciling your local bank statement.</para>
+
+      <para>If you have different currencies you might have to manually convert
+      the amounts from one currency to another while you reconcile the
+      accounts.</para>
     </sect2>
   </sect1>
 
-  <sect1 id="currency_reconcile1">
-    <title>Reconciling Statements in a Foreign Currency</title>
-
-    <para>Reconciling foreign statement are done in the same manner as when
-    you reconcile your local bank statement. If you have created a Chart of
-    Accounts structure which allows you to have the same currency per account
-    as your statement, it is actually exactly the same as
-    reconciling your local bank statement.</para>
+  <sect1 id="currency_trading_accts">
+    <title>Automatically Recording Currency Transactions using Trading Accounts</title>
+
+    <para>Trading accounts were introduced as a feature from  &app; version 2.3.14. The
+    conceptual basis for trading accounts is described in
+    <ulink type="http" url="http://www.mathstat.dal.ca/~selinger/accounting/tutorial.html">
+    this tutorial by Peter Selinger</ulink>. A second tutorial
+    <ulink type="http" url="http://www.mathstat.dal.ca/~selinger/accounting/gnucash.html">
+    by Peter Selinger</ulink> describes the manual creation and use of trading accounts in &app;
+    prior to version 2.3.14.</para>
+
```

Comment:
1. I think this section belongs in the intro part (before currency_manual), so the user can decide where to continue.

1. I am missing a warning about potential pitfalls as decribed by Selinger.